graveler/ref/manager_test.go

```diff
@@ -610,6 +611,37 @@ func (f *fakeAddressProvider) ContentAddress(_ ident.Identifiable) string {
 	return res
 }
 
+func TestConsistentCommitIdentity(t *testing.T) {
+	hex := ident.NewHexAddressProvider()
+
+	commit := graveler.Commit{
+		Committer:    "some-committer",
+		Message:      "I just committed",
+		MetaRangeID:  "123456789987654321",
+		CreationDate: time.Date(2021, time.January, 24, 15, 10, 11, 0, time.UTC),
```

Comment:
Can we also have some microseconds, even nanoseconds, on this time?

Reply:
Yes you can.